### ISSUE TITLE
deno 1.20.3 breaks vr

### ISSUE THREAD

**Issue by ayakovlenko** [jurassiscripts/velociraptor#111]
```
$ vr

error: TS2769 [ERROR]: No overload matches this call.
  Overload 1 of 2, '(input: string | Request, init?: RequestInit | undefined): Promise<Response>', gave the following error.
    Argument of type 'string | URL | Request' is not assignable to parameter of type 'string | Request'.
      Type 'URL' is not assignable to type 'string | Request'.  Overload 2 of 2, '(input: URL, init?: RequestInit | undefined): Promise<Response>', gave the following error.
    Argument of type 'string | URL | Request' is not assignable to parameter of type 'URL'.
      Type 'string' is not assignable to type 'URL'.
  const promise = fetch(url, { signal: controller.signal });

    at https://arweave.net/e-arTRJlkythqLWvvPw09-2ZJU1rIS3l3asHysflhK0/lib/utilities/utils.ts:13:25
```

```
$ deno --version

deno 1.20.3 (release, aarch64-apple-darwin)
v8 10.0.139.6
typescript 4.6.2
```

**Comment by ayakovlenko** [comment]
Looks like the issue is caused by `hatcher` dependency:

```
$ curl -v https://x.nest.land/hatcher@0.10.2/mod.ts

*   Trying 174.138.47.182:443...
* Connected to x.nest.land (174.138.47.182) port 443 (#0)
...
Found. Redirecting to https://arweave.net/e-arTRJlkythqLWvvPw09-2ZJU1rIS3l3asHysflhK0/mod.ts
```

https://github.com/nestdotland/hatcher/blob/628d4c0b518d0628af90f6b8ea6aef57aad45aa1/lib/utilities/utils.ts#L13

Caused by denoland/deno#14113.

**Comment by ayakovlenko** [comment]
Fixed in denoland/deno#14134.
